plugins/pixart-rf/pixart-rf.quirk

```diff
@@ -1,3 +1,7 @@
-# pxi-hid
-[DeviceInstanceId=HIDRAW\VEN_093A&DEV_2801]
+# Pixart Devcie
+[DeviceInstanceId=HIDRAW\VEN_093A]
+Plugin = pixart_rf
+
+# Primax Devcie
+[DeviceInstanceId=HIDRAW\VEN_0461]
 Plugin = pixart_rf
```

Comment:
All Primax devices?!

Reply:
No, Just part of devices

Reply:
So you need to include the `DeviceInstanceId=HIDRAW\VEN_0461&PID_XXXX` values too.

Reply:
We will detect the device in the plugin, if the passed by `fu_pxi_device_fw_ota_init ` and `fu_pxi_device_fw_get_info ` successfully. I am not sure that the PID  is necessary.

Reply:
Is every Primax device ever created going to respond well to being opened and queried? I'd be much more happy with an explicit list...

Reply:
Still this.

Reply:
I will add each pid into quirks